### ISSUE TITLE
More documentation of accuracy

### ISSUE THREAD

**Issue by itrharrison** [alessiospuriomancini/cosmopower#11]
When implementing CosmoPower for SOLiket (see [here](https://github.com/simonsobs/SOLikeT/pull/76)) we evaluated the accuracy against CAMB and CLASS runs, finding O(1.e-2) accuracy.

It would be great if this expectation could have been compared to something documented here, in order to understand whether out implementation had succeeded.

I am aware of the accuracy plots in the paper, but because they require the sigma_ell corresponding to SO they are not as intuitive or easy to generate.

**Comment by dpiras** [comment]
These are the accuracy plots, _before_ dividing by sigma_ell given by SO, for TT; they are typically within O(1e-3). I attach 3 samples here, hope you can read the labels. Happy to answer any other questions!
![image](https://user-images.githubusercontent.com/25639122/198324045-3aee98bb-5488-4b5f-b46d-e997c437643e.png)

**Comment by itrharrison** [comment]
Brilliant, thank you! We are using the pre-trained networks:
```
    network_path: str = "CP_paper/CMB"
    cmb_tt_nn_filename: str = "cmb_TT_NN"
    cmb_te_pcaplusnn_filename: str = "cmb_TE_PCAplusNN"
    cmb_ee_nn_filename: str = "cmb_EE_NN"
```
and when comparing to a CLASS-generated spectra we also got O(1.e-3) agreement 👍 . I'm happy with being able to find those plots again in this Issue, but I guess they could also be added in the `getting_started` notebooks?

**Comment by alessiospuriomancini** [comment]
thanks for the suggestion @itrharrison, will do!
